.github/workflows/concrete_python_release.yml

```diff
@@ -138,7 +141,15 @@ jobs:
           
           pip install $CONCRETE_PYTHON/dist/*manylinux*.whl
           pip install -r $CONCRETE_PYTHON/requirements.dev.txt
+
+          expected_version=$(cat "$CONCRETE_PYTHON/version.txt")
+          actual_version=$(python -c "from concrete import fhe; print(f'__version__ = \"{fhe.__version__}\"')")
           
+          if [ $actual_version != $expected_version ]; then
```

Comment:
doesn't need to be 

 if [ "$actual_version" != "$expected_version" ]; then
 
 ?

Reply:
Yeap, fixed.